domains/transfer/Transfer.API/API.IntegrationTests/SwaggerTests.GetSwaggerDoc_AppStarted_NoChangesAccordingToSnapshot.verified.txt

```diff
@@ -671,6 +700,15 @@
           }
         },
         "additionalProperties": false
+      },
+      "TransferAutomationStatus": {
+        "type": "object",
+        "properties": {
+          "status": {
+            "type": "string"
```

Comment:
Why is the status a magic string? Why not use something more type safe e.g. a boolean value called "healthy"; when true it is healthy, when false it is not healty. 
Or potentially use http status codes for this